ThunderstoreCLI/Utils/Write.cs

```diff
@@ -1,16 +1,33 @@
-using System;
-using System.Linq;
 using static Crayon.Output;
 
 namespace ThunderstoreCLI
 {
     public static class Write
     {
-        private static void _Error(string msg) => Console.WriteLine(Red(msg));
-        private static void _Light(string msg) => Console.WriteLine(Dim(msg));
-        private static void _Regular(string msg) => Console.WriteLine(msg);
-        private static void _Success(string msg) => Console.WriteLine(Green(msg));
-        private static void _Warn(string msg) => Console.WriteLine(Yellow(msg));
+        private static void _Error(string msg)
+        {
+            Console.WriteLine(Red(msg));
+        }
```

Comment:
It seems odd to move away from lambda style method declarations when they fit on one line just fine

Reply:
I have changed this rule in a0ca2e82db8a366a5f52c9a1d910234cd003bb5f